Makefile

```
@@ -37,6 +37,11 @@ endif
 collector: builder
 	make -C collector container/bin/collector
 
+.PHONY: connscrape
+connscrape:
+	make -C collector connscrape ||\
+		( .openshift-ci/slack/notify-if-needed.sh "connscrape" $$? )
```

Notification are specific to CI, we should probably not do this as part of the make target so we can safely run it locally.
```suggestion
	make -C collector connscrape
```